<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="minimumSize">
   <size>
    <width>312</width>
    <height>77</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>312</width>
    <height>77</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Application Main Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="1">
      <widget class="QPushButton" name="FindCustomerButton">
       <property name="toolTip">
        <string>Do a search for Customers</string>
       </property>
       <property name="text">
        <string>Find Customer</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="RemoveCustomerButton">
       <property name="text">
        <string>Remove Customer</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="ShowCustomersButton">
       <property name="text">
        <string>ShowCustomers</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="AddCustomerButton">
       <property name="toolTip">
        <string>Add a customer to the Database</string>
       </property>
       <property name="text">
        <string>Add Customer</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="AboutButton">
       <property name="text">
        <string>About</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="QuitButton">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
